src/ceph.in

```diff
@@ -183,6 +183,67 @@ def mdsids():
         l.append(mdsdict['name'])
     return l
 
+def mgrids():
+    ret, outbuf, outs = json_command(cluster_handle, prefix='mgr dump',
+                                     argdict={'format': 'json'})
+    if ret == -errno.EINVAL:
+        # try old mon
+        ret, outbuf, outs = send_command(cluster_handle,
+                                         cmd=['mgr', 'dump', '--format=json'])
+    if ret:
+        raise RuntimeError('Can\'t contact mon for mgr list')
+
+    d = json.loads(outbuf.decode('utf-8'))
+    l = []
+    l.append(d['active_name'])
+    for i in d['standbys']:
+        l.append(i['name'])  
+    return l
+
+def validate_target(target):
+    """
+      this function will return true iff target is a correct
+      target, such as mon.a/osd.2/mds.a/mgr.
+
+      target: str
+      return: bool, or raise RuntimeError
+    """
+    _target = target.split('.')
+    ret = False
+
+    if len(_target) == 2:
+        # for case "service.id"
+        service_name, service_id = _target[0], _target[1]
+        exist_ids = []
+        if service_name == "mon":
+            exist_ids = monids()
+        elif service_name == "osd":
+            exist_ids = osdids()
+        elif service_name == "mds":
+            exist_ids = mdsids()
+        elif service_name == "mgr":
+            exist_ids = mgrids()
+        else:
+          if verbose:
+            print('WARN: {0} is not a legal service name, should be one of mon/osd/mds/mgr'.format(service_name),
+                  file=sys.stderr)
+          return False
+
+        if service_id in exist_ids:
+            ret = True
+
+        if not ret and verbose:
+            print('WARN: the service id you provided does not exist. service id should '
+                  'be one of {0}.'.format('/'.join(exist_ids)), file=sys.stderr)
+
+    elif len(_target) == 1 and _target[0] == "mgr":
```

Comment:
```
➜  build git:(wip-19885-mgr-help) ✗ ceph tell mon.a status

    cluster 0f638af9-bf50-414d-85d0-3f72a8664d91
     health HEALTH_OK
     monmap e2: 3 mons at {a=127.0.0.1:40000/0,b=127.0.0.1:40001/0,c=127.0.0.1:40002/0}
            election epoch 10, quorum 0,1,2 a,b,c
      fsmap e7: 1/1/1 up {0=a=up:active}, 2 up:standby
        mgr active: x 
     osdmap e11: 3 osds: 3 up, 3 in
      pgmap v22733: 24 pgs, 3 pools, 2246 bytes data, 21 objects
            170 GB used, 242 GB / 413 GB avail
                  24 active+clean
➜  build git:(wip-19885-mgr-help) ✗ ceph tell mon status  
error handling command target: CephName: no . in mon
```
I tested it in my PC, it seems that `mon` is not allowed in `tell` command. maybe we should only support `mgr`  here. I'm pleasure to add `mon` when we have fixed the problem `error handling command target: CephName: no . in mon`. how about?